pyqir/src/values.rs

```diff
@@ -385,6 +409,40 @@ impl From<Linkage> for inkwell::module::Linkage {
     }
 }
 
+#[pyclass(extends = Constant)]
+pub(crate) struct ConstantExpr;
+
+#[pymethods]
+impl ConstantExpr {
+    /// Creates a `getelementptr` (GEP) constant expression.
+    ///
+    /// :param Value value: The aggregate value.
+    /// :param Sequence[Value] indices: The indices of the element.
+    /// :param bool inbounds: Whether to create an in-bounds GEP.
+    /// :returns: The GEP constant expression.
+    /// :rtype: ConstantExpr
+    #[staticmethod]
+    #[allow(clippy::needless_pass_by_value)]
```

Comment:
Please add a test for inbounds == false

Reply:
For some reason, LLVM doesn't seem to care if I use inbounds or not - it emits `getelementptr inbounds` no matter what... so I just removed the parameter.